src/mpp/Dec.hpp

```diff
@@ -1172,6 +1172,27 @@ bool jump_read_key(BUF& buf, T... t)
 	return jump_find_key<PAIRS, FAMILY, PATH>(val, IS, buf, t...);
 }
 
+template<compact::Family FAMILY, size_t SUBRULE,
+	 class DST, class BUF, class = void>
+struct is_optional_or_variant_readable_in_place : std::false_type {};
+
+template<compact::Family FAMILY, size_t SUBRULE,
+	 class DST, class BUF>
+struct is_optional_or_variant_readable_in_place
```

Comment:
It should be about arbitrary object, not only `optional` or `variant`.

Reply:
Fixed.